autogpt_platform/backend/backend/server/rest_api.py

```diff
@@ -65,9 +67,13 @@ def run_service(self):
         if self._test_dependency_overrides:
             app.dependency_overrides.update(self._test_dependency_overrides)
 
+        logger.info(
+            f"FastAPI CORS allow origins: {Config().backend_cors_allow_origins}"
+        )
```

Comment:
Why do we need a log statement for this if it's configured by the admin?

Reply:
I've changed it to a debug message